AutoRest/Generators/CSharp/CSharp/Templates/MethodTemplate.cshtml

```diff
@@ -228,7 +228,7 @@ else
 
     HttpStatusCode _statusCode = _httpResponse.StatusCode;
     cancellationToken.ThrowIfCancellationRequested();
-    string _responseContent = null;
+    string _responseContent = await _httpResponse.Content.ReadAsStringAsync().ConfigureAwait(false);
```

Comment:
Good point. This was the same code generated for the 200 case, but I guess that was for a valid model instead of a stream. I'll investigate an improved fix.